### ISSUE TITLE
Official OpenVSX Release

### ISSUE THREAD

**Issue by trymeouteh** [bash-lsp/bash-language-server#1180]
Please claim the namespace for this extension on OpenVSX

https://open-vsx.org/extension/mads-hartmann/bash-ide-vscode

**Comment by shyguyCreate** [comment]
I agree, the extension in OpenVSX has become very outdated and misses the new features like support for `shfmt`.

**Comment by shyguyCreate** [comment]
I have open a [PR](https://github.com/open-vsx/publish-extensions/pull/791) to update the extension provided by openvsx. Let's see if that works.
